### PR TITLE
Update the copy on the sign up complete page

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

N/A

### Context and changes

https://docs.google.com/document/d/1zos7X-rsD8MH08GSw0eXQ4n9z11VLppS_fu9jPBvb_I/edit

### CODE REVIEW

**Comment by MylesJarvis** [comment]
@Tomcowan1 - there's a few more other page updates in here, not just confirmation page. I think definitely run past Sarah BH now just to confirm they're happy, especially on the stuff about WhatsApp etc.

**Comment by MylesJarvis** [comment]
Not going to approve for now as don't think I can give it the time it needs before I need to head offline - will pick up Monday

**Comment by Tomcowan1** [comment]
Thanks Myles - agreed the Whatsapp messages with Sarah BH this week - sorry I should have mentioned. Let's chat when you're back next week.

File: app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <%= f.govuk_fieldset legend: { text: 'You told us you have an equivalent degree and live overseas' } do %>
 
-<p>We will need to call you to check your degree. You must have the details of your overseas qualifications when we contact you.</p>
-<p>In the meantime, you can call us on Freephone <%= link_to("0800 389 2500", "tel://08003892500") %> between 8.30am and 5.30pm Monday to Friday.</p>
+<p>We will need to call you to check your degree before you can get an adviser.</p>
+
+<p>We will call you between 8.30am and 5.30pm (UK time).</p>
```
Comment (MylesJarvis) [inline app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:5]: just to check @Tomcowan1, what's the logic for removing option to call themselves in meantime?

File: app/views/teacher_training_adviser/steps/completed.html.erb [hunk 2]
```diff
@@ -21,28 +21,36 @@
...
-        <p>
-          You’ll get an email from us soon which will outline the next steps for you to talk to an adviser.
-        </p>
-        <p>
-          You must respond to the email within 6 weeks of receiving it otherwise you will have to sign up again.
-        </p>
+
+        <p>An adviser will email you within 3 working days.</p>
+
+        <p>You need to reply to the email to be assigned an adviser.</p>
```
Comment (MylesJarvis) [inline app/views/teacher_training_adviser/steps/completed.html.erb:27]: Is this agreed with Sarah too @Tomcowan1?

Comment (Tomcowan1) [reply app/views/teacher_training_adviser/steps/completed.html.erb:27]: Yes she did :-)

File: app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb (hunk 1 above)
Comment (Tomcowan1) [reply app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:5]: Good question @MylesJarvis. When I was reading through the original I thought it was a bit confusing - we're saying we'll call people, but then telling them to call us at the same time. Much easier to have one or the other. 

Plus, we used to say "have the details ready when we contact you" which was a bit odd - we shouldn't expect people to carry them around until we call.

Comment (Tomcowan1) [reply app/views/teacher_training_adviser/steps/_overseas_time_zone.html.erb:5]: Hi @MylesJarvis I've edited this one and am checking it again with Sarah BH. The other two changes (confirmation page / telephone number) are good to go, if they can be deployed separately?

**Review by MylesJarvis — APPROVED** [review]

**Review by Tomcowan1 — APPROVED** [review]

**Pull request merged**